04-account-kit-manage-permissions/index.ts

```diff
@@ -0,0 +1,48 @@
+// SDK exposes all you need
+import { Chains, PublicKey, SigningRequest, Permission } from '@[SOMETHING]kit'
+import { Account } from '@[SOMETHING]kit/account'
+
+async function main() {
+    // Loads account
+    const account: Account = await Account.load("teamgreymass", Chains.EOS)
+
+    // Manage account Permissions
+
+    // Get a permission
+    const permission: Permission = account.permissions.get("active")
+
+    // Get its public keys
+    const publicKeys: PublicKey[] = permission.keys
+
+    // Get its accounts
+    const accounts: Account[] = permission.accounts
+
+    // Add a new permission
+    const addPermissionEsr: SigningRequest = account.permissions.add('active', PublicKey.from('PUB_K1_5rcqJ8eNgNXooYyz9PTX2a2zZBktWaWezhXYGN2hrXWqfHSWYY'))
+
+    // Update a new permission
+    const updatePermissionEsr: SigningRequest = account.permissions.get('active').update(PublicKey.from('PUB_K1_5rcqJ8eNgNXooYyz9PTX2a2zZBktWaWezhXYGN2hrXWqfHSWYY'))
+
+    // Remove a permission
+    const removePermissionEsr: SigningRequest = account.permissions.get('active').remove()
```

Comment:
Should we require `.get('active').remove()` or just `.remove('active')`? 🤔

Reply:
The reason why I like the `.get('active').remove()` option is because it leverages the same `get` method which is used to get the specific permission. That's one of those things where I'd be happy to go either ay though as long as we are consistent across the sdk.

Reply:
We talked a bit on the call about this, but the reason to avoid chaining methods like `.get('active').remove()` is because `.get('active')` should probably return an instance of an `Authority`.

https://github.com/greymass/eosio-core/blob/fdc99cbad55738ea1912135478c1ae472682df3a/src/chain/authority.ts#L45-L103

This `Authority` object isn't going to have a `.remove()` method on it, or know the context of where it came from - unless we create some new type of wrapper for `Authority` that's aware of its context.